sdk/communication/Azure.Communication.Sms/src/SmsClient.cs

```diff
@@ -117,37 +121,50 @@ public virtual async Task<Response<SendSmsResponse>> SendAsync(PhoneNumberIdenti
         /// <param name="from">The sender's phone number that is owned by the authenticated account.</param>
         /// <param name="to">The recipient's phone number.</param>
         /// <param name="message">The contents of the message that will be sent to the recipient. The allowable content is defined by RFC 5724. If the message has more than 160 characters, the server will split it into multiple SMSs automatically.</param>
-        /// <param name="sendSmsOptions">Optional configuration for sending SMS messages.</param>
+        /// <param name="options">Optional configuration for sending SMS messages.</param>
         /// <param name="cancellationToken">The cancellation token for the underlying request.</param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
         /// <exception cref="ArgumentNullException"><paramref name="from"/> is null.</exception>
         /// <exception cref="ArgumentNullException"><paramref name="to"/> is null.</exception>
         /// <exception cref="ArgumentNullException"><paramref name="message"/> is null.</exception>
-        public virtual Response<SendSmsResponse> Send(PhoneNumberIdentifier from, PhoneNumberIdentifier to, string message, SendSmsOptions sendSmsOptions = null, CancellationToken cancellationToken = default)
+        public virtual Response<SmsSendResult> Send(string from, string to, string message, SmsSendOptions options = default, CancellationToken cancellationToken = default)
```

Comment:
perhaps rename options to smsSendOptions? it's consistent with the previous convention too